<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #1E1E1E;
    color: #E0E0E0;
}
</string>
  </property>
  <widget class="QPushButton" name="ajouter_v">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #2E2E2E;  /* Fond de la boîte de dialogue */
    border: none;  /* Pas de bordure autour de la boîte */
    padding: 10px;  /* Espacement autour des boutons */
    border-radius: 8px;  /* Coins arrondis */
}

QPushButton {
    background-color: #6A5ACD;  
    color: white; 
    padding: 8px 15px; 
    border: 1px solid #483D8B; 
    border-radius: 6px; 
    font-size: 14px; 
    font-family: 'Segoe UI', sans-serif;  
 
}

QPushButton:hover {
    background-color: #7B68EE;  /* Couleur au survol */
}

QPushButton:pressed {
    background-color: #483D8B;  /* Couleur lors du clic */
}

QPushButton:disabled {
    background-color: #555555;  /* Couleur des boutons désactivés */
    color: #999999;  /* Couleur du texte des boutons désactivés */
}
</string>
   </property>
   <property name="text">
    <string>Enregistrer la ville</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ville_saisie">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    QLineEdit {
        border-radius: 8px;
        border: 1px solid #444;
        padding: 8px;
        font-size: 14px;
        background-color: #2B2B2B;
        color: #FFFFFF;
    }
    QLineEdit:focus {
        border-color: #1E88E5;
        background-color: #333;
        color: #FFFFFF;
    }
    </string>
   </property>
   <property name="placeholderText">
    <string>Saisir la ville à ajouter</string>
   </property>
  </widget>
  <widget class="QLabel" name="etat_p">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;
font-weight: bold;
color: #008000;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etat_n">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;
font-weight: bold;
color: #FF6F61;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
